lexer.go

```diff
@@ -175,6 +189,7 @@ func (s *Scanner) Lex(v *yySymType) int {
 		tok == stringLit &&
 		s.r.s[v.offset] == '"' {
 		tok = identifier
+		v.ident = s.tryDecodeToUTF8(v.ident)
```

Comment:
there are many codes like this, they all convert `v.ident` to its utf8 version if the token is the identifier, can we just write it at L214 and remove the others.